samples/java-protobuf-valueentity-counter/src/it/java/com/example/CounterIntegrationTest.java

```diff
@@ -20,57 +23,59 @@
-      CounterApi.CurrentCounter reply = client.getCurrentCounter(CounterApi.GetCounter.newBuilder().setCounterId(entityId).build())
-              .toCompletableFuture().get(5, SECONDS);
-      assertThat(reply.getValue(), is(42));
+    String entityId = "new-id";
+    client.increase(CounterApi.IncreaseValue.newBuilder().setCounterId(entityId).setValue(42).build()) // <3>
+        .toCompletableFuture().get(5, SECONDS);
+    CounterApi.CurrentCounter reply = client.getCurrentCounter(CounterApi.GetCounter.newBuilder().setCounterId(entityId).build())
+        .toCompletableFuture().get(5, SECONDS);
+    assertThat(reply.getValue(), is(42));
   }
   // end::sample-it-test[]
 
   @Test
   public void increase() throws Exception {
-      String entityId = "another-id";
-      client.increase(CounterApi.IncreaseValue.newBuilder().setCounterId(entityId).setValue(42).build())
-               .toCompletableFuture().get(5, SECONDS);
-      client.increase(CounterApi.IncreaseValue.newBuilder().setCounterId(entityId).setValue(27).build())
-               .toCompletableFuture().get(5, SECONDS);
-      CounterApi.CurrentCounter reply = client.getCurrentCounter(CounterApi.GetCounter.newBuilder().setCounterId(entityId).build())
-              .toCompletableFuture().get(5, SECONDS);
-      assertThat(reply.getValue(), is(69));
+    String entityId = "another-id";
+    client.increase(CounterApi.IncreaseValue.newBuilder().setCounterId(entityId).setValue(42).build())
+        .toCompletableFuture().get(5, SECONDS);
+    client.increase(CounterApi.IncreaseValue.newBuilder().setCounterId(entityId).setValue(27).build())
+        .toCompletableFuture().get(5, SECONDS);
+    CounterApi.CurrentCounter reply = client.getCurrentCounter(CounterApi.GetCounter.newBuilder().setCounterId(entityId).build())
+        .toCompletableFuture().get(5, SECONDS);
+    assertThat(reply.getValue(), is(69));
   }
 
-    @Test
-    public void validateGrpcStatusCode() throws Exception {
-        try {
-            client.increaseWithConditional(CounterApi.IncreaseValue.newBuilder().setCounterId("new-id").setValue(-10).build())
-                    .toCompletableFuture().get(5, SECONDS);
-            Assert.fail("Previous method should be invalid and throw exception");
-        } catch (ExecutionException e) {
-            var suppressed = (StatusRuntimeException) e.getCause();
-            assertThat(suppressed.getStatus().getCode(), is(Status.Code.INVALID_ARGUMENT));
-        }
+  @Test
+  public void validateGrpcStatusCode() throws Exception {
+    try {
+      client.increaseWithConditional(CounterApi.IncreaseValue.newBuilder().setCounterId("new-id").setValue(-10).build())
+          .toCompletableFuture().get(5, SECONDS);
+      Assert.fail("Previous method should be invalid and throw exception");
+    } catch (ExecutionException e) {
+      var suppressed = (StatusRuntimeException) e.getCause();
+      assertThat(suppressed.getStatus().getCode(), is(Status.Code.INVALID_ARGUMENT));
```

Comment:
Most changes on this file are formatting changes to align with the remaining files (2 spaces instead of 4)